all.yml

```diff
@@ -58,7 +58,7 @@
 - hosts: bahmni-reports-db,bahmni-reports-db-slave
   sudo: yes
   roles:
-     - { role: mysql }
+     - { role: mysql, when: mysql_role_ran is not defined or mysql_role_ran != true }
```

Comment:
why this task is not there for bahmni-emr-db,bahmni-emr-db-slave ??

Reply:
bahmni-emr-db, bahmni-emr-db-slave will be installed prior to bahmni-reports.
So MySql password will be updated in when mysql play gets run for the very first time during installation of bahmni-emr-db,bahmni-emr-db-slave 
During installation of bahmni-reports mysql tasks were being run for the second time 
and failing because mysql password is already updated to new password specified in setup.yml
So put conditional check there.